api/v1alpha1/privateloadzone_types.go

```diff
@@ -34,6 +34,7 @@ import (
 type PrivateLoadZoneSpec struct {
 	Token              string                        `json:"token"`
 	Resources          corev1.ResourceRequirements   `json:"resources"`
+	Tolerations        []corev1.Toleration           `json:"tolerations"`
```

Comment:
```suggestion
	Tolerations        []corev1.Toleration           `json:"tolerations,omitempty"`
```
This is needed to make `spec.tolerations` an optional field in PLZ. 
`make manifests` will need to be re-run afterwards.

Reply:
Thanks! Pushed the fix - hopefully I'll have the go-ahead from my company in a couple of days 🤞